### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dmikusa-pivotal**
Update the TIPs section.

### CODE REVIEW

**Comment by dmikusa-pivotal** [comment]
Don't merge this until the following PRs have been merged:

- https://github.com/paketo-community/cargo/pull/146
- https://github.com/paketo-community/rustup/pull/45

And a new release has been cut.

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**
